code/hud/hudconfig.cpp

```diff
@@ -349,6 +288,13 @@ struct HC_gauge_region	HC_gauge_regions[GR_NUM_RESOLUTIONS][NUM_HUD_GAUGES] =
 	}
 };
 
+struct gauge_coords {
+	int x;
+	int y;
+};
+
+static SCP_vector<gauge_coords> HC_gauge_coords;
```

Comment:
keyword `static` is probably moot for members defined the .cpp's scope.  (Don't have to change)